extensions/sql-database-projects/src/projectProvider/projectAssets.ts

```diff
@@ -1,20 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the Source EULA. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-import * as path from 'path';
-
-export interface ProjectAssets {
-	readmeFolder?: string
```

Comment:
All the files in the resources/projectAssests folder can also be removed, right?

Reply:
correct I delete the folder as well